lib/definitions/errors.js

```diff
@@ -5,6 +5,12 @@ module.exports = {
     message: 'Deployment to maven failed.',
     details: `The deployment to maven failed for an unknown reason.
 
+Please check the logs on the CI server to see what happened.`
+  }),
+  EMAVENPACKAGE: () => ({
+    message: 'Packaging with maven failed.',
+    details: `Maven failed package the project for an unknown reason.
```

Comment:
Possible typo. Should this be `Maven failed to package the project for an unknown reason.`?